app/src/main/java/com/github/se/polyfit/ui/utils/GraphData.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.se.polyfit.ui.utils
+
+data class GraphData(val kCal: Double, val day: Int, val month: String, val weight: Double) {
```

Comment:
it would be a good idea to write unit tests for this

Reply:
will do when I implement the Data as it will probably modified a little to accommodate